pkg/kubectl/cmd/get.go

```diff
@@ -94,6 +95,23 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		return err
 	}
 
+	if len(args) == 0 {
+		fmt.Fprint(out, `
+You must specify the type of resource to get. Valid resource types include:
+   * pods (aka 'po')
+   * replicationcontrollers (aka 'rc')
+   * services
```

Comment:
@hurf yeah, looking at the commit where the comment was added (https://github.com/GoogleCloudPlatform/kubernetes/commit/09dcec57fb6a2956b815efbd448afea75529a29a), it seems like that was the intent